projects/cdk/utils/array/filter.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Returns a new filtered array. The search is performed by the properties specified in the keys.
+ *
+ * @param list Array of objects
+ * @param search Search string
+ * @param keys Specified keys for search
+ *
+ * @returns Filtered array of objects
+ */
+export const tuiFilterList = <T>(
```

Comment:
I don't think we want this as generalized helper, otherwise it might start growing with customizations like `stringify`. Let's just check 2 keys locally in the component using `TUI_DEFAULT_MATCHER`